ecommerce/certprep/static/sass/partials/views/_basket.scss

```diff
@@ -0,0 +1,703 @@
+.basket {
```

Comment:
Are you sure you need al these definitions? @sergivalero20

Reply:
Done!
I removed unnecessary rules.